### ISSUE TITLE
How to send using sendgrid template?

### ISSUE THREAD

**Issue by yagobski** [Cellarise/loopback-connector-sendgrid#5]
I want to use sendgrid template. Do you have any idea how to do that?

something like : 

```
user.app.models.Email.send({
                            to: usr.email,
                            from: config.adminEmail,
                            subject: "Email Verification",
                            html: " ",
                            "x-smtpapi": {
                                "sub": {
                                    "*|USERNAME|*": [
                                        usr.username
                                    ],
                                    "*|URL|*": [
                                        url
                                    ]
                                },
                                "filters": {
                                    "templates": {
                                        "settings": {
                                            "enable": 1,
                                            "template_id": "342986ef-4ca7-45c3-9b69-XXXXb20"
                                        }
                                    }
                                },
                                category: "user"
                            }


                        }
```

**Comment by JohnMBarry** [comment]
Sorry for taking so long to reply.  Filters should be configurable in datasources.json or when creating the sendgrid datasource.

`{
    "sendgrid": {
        "connector": "loopback-connector-sendgrid",
        "api_user": '[your username here]'
        "api_key": '[your password here]',
        "filters": {
                'templates': {
                    "settings": {
                         'enable': 1,
                         'template_id' : '09c6ab89-9157-4710-8ca4-db7927c631d6',
                     }
                }
        }
    }
}`

**Comment by chuckcarpenter** [comment]
@JohnMBarry this didn't work as is, I have a fix in #6 though that makes this work as you'd suggested. Just doesn't get passed to the SendGrid setHeaders method since `this.filters` is never truthy.
